recipe/meta.yaml

```diff
@@ -36,11 +36,11 @@ requirements:
     - pandas >=1.4.1,<1.6
     - scikit-learn >=1.0,<1.3
     - networkx >=3.0,<4
-    - pytorch >=1.13,<2.1
+    - pytorch >=2.0,<2.1
```

Comment:
change it back to '>=1.13' so that autogluon can be installed in env with pytorch <2.0 and newer versions.